lib/index.js

```diff
@@ -49,7 +49,7 @@
       return tag
     }
 
-    return str.replace(/\[(\d+;)?(\d+)*m/g, function(match, b1, b2) {
+    return str.replace(/\[(\d+;)?(\d+){0,2}m/g, function(match, b1, b2) {
```

Comment:
How did you come with a pattern of 0 to 2 length restriction?
Are you able to provide user acceptance testing to validate that there will be no breaking change?